### PR TITLE
\JSI - android - part 3

### PR DESCRIPTION
**Pull request by radex**

### CODE REVIEW

File: native/android-jsi/src/main/java/com/nozbe/watermelondb/jsi/WatermelonJSI.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+// Public interface to JSI-based Watermelon
+public class WatermelonJSI {
+    public static void install(Application context, long javaScriptContextHolder) {
+        JSIInstaller.install(context, javaScriptContextHolder);
+    }
+
+    public static void onTrimMemory(int level) {
+      // TODO: Unimplemented
+    }
```
Comment (rozPierog) [inline native/android-jsi/src/main/java/com/nozbe/watermelondb/jsi/WatermelonJSI.java:13]: What do you want to do in `onTrimMemory` later?

Comment (radex) [reply native/android-jsi/src/main/java/com/nozbe/watermelondb/jsi/WatermelonJSI.java:13]: well… trim… memory :) remove sqlite caches, stuff like that.

File: .travis.yml [hunk 2]
```diff
@@ -65,6 +65,8 @@ matrix:
         - emulator -avd test -no-skin -no-window &
       install:
         - yarn
+        # In case stuff breaks, look at:
+        # https://github.com/android/ndk-samples/blob/master/.travis.yml#L16
         - echo y | sdkmanager 'ndk-bundle'
         - echo y | sdkmanager --channel=3 --channel=1 "cmake;3.10.2.4988404"
         - echo y | sdkmanager "lldb;3.1"
```
[the remark this answers is not in the document]
Comment (radex) [reply .travis.yml:72]: is it worth it? ¯\_(ツ)_/¯

[the remark this answers is not in the document]
Comment (rozPierog) [reply .travis.yml:72]: Yes it is. Check Active PRs, only iOS builds are broken. Android worked fine. We need _something_ that tells us if Watermelon still works - you can also comment out iOS part when you are doing it - it's useless right now

[the remark this answers is not in the document]
Comment (radex) [reply .travis.yml:72]: shit, fine.

**Comment by radex** [comment]
I believe it should pass now :)

**Comment by radex** [comment]
fuck, something nasty's here. passes on my machine beautifully

**Pull request merged**